cpp/libcugraph_etl/cmake/thirdparty/get_cudf.cmake

```diff
@@ -1,49 +1,53 @@
-#=============================================================================
+# =============================================================================
 # Copyright (c) 2021-2025, NVIDIA CORPORATION.
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
 #
-#     http://www.apache.org/licenses/LICENSE-2.0
+# http://www.apache.org/licenses/LICENSE-2.0
 #
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#=============================================================================
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
 
 function(find_and_configure_cudf)
 
-    set(oneValueArgs VERSION FORK PINNED_TAG)
-    cmake_parse_arguments(PKG "" "${oneValueArgs}" "" ${ARGN} )
-
-    rapids_cpm_find(cudf ${PKG_VERSION}
-      GLOBAL_TARGETS      cudf::cudf
-      BUILD_EXPORT_SET    cugraph_etl-exports
-      INSTALL_EXPORT_SET  cugraph_etl-exports
-        CPM_ARGS
-            GIT_REPOSITORY https://github.com/${PKG_FORK}/cudf.git
-            GIT_TAG        ${PKG_PINNED_TAG}
-            SOURCE_SUBDIR  cpp
-            OPTIONS
-              "BUILD_BENCHMARKS OFF"
-              "BUILD_TESTS OFF"
-    )
-
-    message(VERBOSE "CUGRAPH_ETL: Using CUDF located in ${cudf_SOURCE_DIR}")
+  set(oneValueArgs VERSION FORK PINNED_TAG)
+  cmake_parse_arguments(PKG "" "${oneValueArgs}" "" ${ARGN})
+
+  rapids_cpm_find(
+    cudf
+    ${PKG_VERSION}
+    GLOBAL_TARGETS
+    cudf::cudf
+    BUILD_EXPORT_SET
+    cugraph_etl-exports
+    INSTALL_EXPORT_SET
+    cugraph_etl-exports
+    CPM_ARGS
+    GIT_REPOSITORY
+    https://github.com/${PKG_FORK}/cudf.git
+    GIT_TAG
+    ${PKG_PINNED_TAG}
+    SOURCE_SUBDIR
+    cpp
+    OPTIONS
+    "BUILD_BENCHMARKS OFF"
+    "BUILD_TESTS OFF"
+  )
```

Comment:
This doesn't look right to me, I must have mistake in the config, sorry 😬

Reply:
sorry @gforsyth , let me take this back into draft and reduce the size of this diff. I should have done that from the beginning.

Reply:
All good!

Reply:
Ok I think it's looking better, could you check again?

I:

* told `cmake-format` not to touch comments: https://github.com/rapidsai/cugraph/pull/4889#discussion_r1927723375
* described `rapids_cython_create_modules()` to it so it'd stop trying to squish all the keyword arguments onto 1 line: https://github.com/rapidsai/cugraph/pull/4889#discussion_r1927724090